app/server/tests/api-getCommunitiesByTitle.test.js

```diff
@@ -84,3 +84,19 @@ describe('Testing the get community function with the "Welcome" community after
     expect(response.text).toBe("Forbidden.");
   });
 });
+
+describe('Testing the get community function with the "Welcome" community after logging in, but with null as the token', () => {
```

Comment:
lets keep the description short for this to 'Get Community - Post Login' and maybe keeping the details out in a comment header before line 88.

Thoughts?

Reply:
sure will keep the names shorter 👍

I will go back to the previous test files and rename the tests in my next pull request

Reply:
let me know when you've made the changes to this PR and I'll re-review.